whylog/constraints/verifier.py

```diff
@@ -119,6 +119,14 @@ def _construct_proper_clues_lists(cls, original_clues_lists):
         return clues_lists
 
     @classmethod
+    def _pack_results_for_constraint_or(cls, combination, constraints):
+        return cls._create_investigation_result(
+            [
```

Comment:
this creates a list and `_create_investigation_result` could use an iterator. Use a generator expression instead of list comprehension.